### ISSUE TITLE
Migrate-Domain fails for cluster with error: httpclient: raw req: unexpected status 503

### ISSUE THREAD

**Issue by aaronfranco** [flynn/flynn#3399]
I just launched a new cluster on the latest version. I am now trying to migrate the domain for the cluster but I'm getting a strange error,

httpclient: raw req: unexpected status 503

I tried updating the controller's route with the new domain and SSL cert, but then I get errors in the router that say  CA Authority Error.

Any chance you can help with this?  Please find the debug output attached in a TAR file.

I launched the cluster via the Amazon installer using the CIDR 171.161.0.0/16 and subnet CIDR 171.161.0.0/21

The installation failed a few times when I attempted to import from a backup. I then export each app and imported them into the new cluster.
I then upgraded the software for each app, but they could not log information and none of them would start. So, I then used "Flynn create" to launch a new app for each and migrate the ENV variables. I then deleted all the broken apps. I then moved my domain on Route 53 to my custom domain. Then I tried to "migrate-domain" for the cluster.

Best,

Aaron

**Comment by titanous** [comment]
I don't see the debug info file linked, can you add that? Thanks!

**Comment by aaronfranco** [comment]
[flynn-host-debug.tar.gz](https://github.com/flynn/flynn/files/477590/flynn-host-debug.tar.gz)

**Comment by aaronfranco** [comment]
Okay, 

So, I thought I might have messed something up somewhere, but I tried to launch a new cluster under the CIDR:

172.162.0.0/16 
172.162.0.0/21

Then, as soon as it was done installing I ran the migrate-domain command and go this error:
GET https://controller.api.nimbleparse.com/apps/controller/release: httpclient: raw req: unexpected status 503

**Comment by aaronfranco** [comment]
I get the same error if I install a new cluster without the CIDR adjustments.

GET https://controller.api.nimbleparse.com/apps/controller/release: httpclient: raw req: unexpected status 503

**Comment by titanous** [comment]
What is the output of `flynn cluster`?

**Comment by aaronfranco** [comment]
NAME              CONTROLLER URL                          GIT URL                          DOCKER URL
flynn-1463095935  https://controller.api.nimbleparse.com  https://git.api.nimbleparse.com  https://docker.api.nimbleparse.com
flynn-1473905747  https://controller.m6p4.flynnhub.com    https://git.m6p4.flynnhub.com    (none)
flynn-1473947631  https://controller.qkdq.flynnhub.com    https://git.qkdq.flynnhub.com    (none)
flynn-1474061081  https://controller.9kmw.flynnhub.com    https://git.9kmw.flynnhub.com    (none)
flynn-1474061557  https://controller.k661.flynnhub.com    https://git.k661.flynnhub.com    (none)  (default)

**Comment by titanous** [comment]
Try running `flynn -c flynn-1474061557 migrate-domain api.nimbleparse.com`, it is probably picking up the old cluster URL from the git remote of the directory you are in.

**Comment by aaronfranco** [comment]
```
> flynn -c flynn-1473947631 migrate-domain api.nimbleparse.com
migrate-domain is not a flynn command. See 'flynn help'
```

**Comment by titanous** [comment]
Sorry, I had a typo, add `cluster` before `migrate-domain`

**Comment by aaronfranco** [comment]
```
> flynn -c flynn-1473947631 cluster migrate-domain api.nimbleparse.com
Migrate cluster domain from "qkdq.flynnhub.com" to "api.nimbleparse.com"? (yes/no): yes
Migrating cluster domain (this can take up to 2m0s)...
scheduler process type failed to start, got down job event
```

**Comment by titanous** [comment]
Please grab the cluster debug info again.

**Comment by aaronfranco** [comment]
```
INFO[09-16|23:53:23] uploading logs and debug information to a private, anonymous gist 
INFO[09-16|23:53:23] this may take a while depending on the size of your logs 
INFO[09-16|23:53:23] getting flynn-host logs 
INFO[09-16|23:53:24] getting scheduler state 
INFO[09-16|23:53:24] getting job logs 
INFO[09-16|23:55:50] getting system information 
EROR[09-16|23:55:50] error capturing output of "ps faux"      err="fork/exec /bin/ps: cannot allocate memory"
EROR[09-16|23:55:50] error capturing output of "ifconfig"     err="fork/exec /sbin/ifconfig: cannot allocate memory"
EROR[09-16|23:55:50] error capturing output of "uname -a"     err="fork/exec /bin/uname: cannot allocate memory"
EROR[09-16|23:55:50] error capturing output of "lsb_release -a" err="fork/exec /usr/bin/lsb_release: cannot allocate memory"
EROR[09-16|23:55:50] error capturing output of "date"         err="fork/exec /bin/date: cannot allocate memory"
EROR[09-16|23:55:50] error capturing output of "free -m"      err="fork/exec /usr/bin/free: cannot allocate memory"
EROR[09-16|23:55:50] error capturing output of "df -h"        err="fork/exec /bin/df: cannot allocate memory"
EROR[09-16|23:55:50] error capturing output of "flynn-host version" err="fork/exec /usr/local/bin/flynn-host: cannot allocate memory"
EROR[09-16|23:55:50] error capturing output of "route -n"     err="fork/exec /sbin/route: cannot allocate memory"
EROR[09-16|23:55:50] error capturing output of "iptables -L -v -n --line-numbers" err="fork/exec /sbin/iptables: cannot allocate memory"
INFO[09-16|23:55:50] Total size of 431950999 bytes exceeds gist limit, falling back to tarball. 
INFO[09-16|23:56:11] created tarball containing debug information at /tmp/flynn-host-debug585584996/flynn-host-debug.tar.gz 
```

The File 34MB so I cannot attach it.

**Comment by titanous** [comment]
Please email it to logs@flynn.io

**Comment by lmars** [comment]
@aaronfranco we received the debug info via email but it looks corrupt (it's just filled with zeroes), can you send it again?

**Comment by aaronfranco** [comment]
https://drive.google.com/file/d/0B2wqzHpa-L8TTjJ4dUNjdExqa2s/view?usp=sharing

**Comment by lmars** [comment]
@aaronfranco yes that is the debug info I downloaded and it is just a file filled with zeroes, can you try uploading the debug info again? You may want to double check the file is correct on the host by running `tar tf path/to/flynn-host-debug_latest.tar.gz`.

**Comment by aaronfranco** [comment]
https://www.dropbox.com/s/ccb6birmk5e9gmh/flynn-host-debug_latest.tar.gz?dl=0

**Comment by aaronfranco** [comment]
I also tried manual installation and get the same error running on AWS.

**Comment by aaronfranco** [comment]
Any update on this?

**Comment by titanous** [comment]
I believe this was fixed by #3886.
